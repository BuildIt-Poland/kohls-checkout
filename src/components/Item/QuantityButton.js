import styled from 'styled-components';

import { fontWeightBold, fontLarge } from '../../styles/designTokens';

const QuantityButton = styled.button`
  font-size: ${fontLarge};
  font-weight: ${fontWeightBold};
  background: none;
  box-shadow: none;
  border: 0;
`;

export default QuantityButton;
