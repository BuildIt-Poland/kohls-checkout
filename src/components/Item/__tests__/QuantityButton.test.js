import React from 'react';
import { create } from 'react-test-renderer';

import QuantityButton from '../QuantityButton';

describe('COMPONENT - QuantityButton', () => {
  it('renders QuantityButton component', () => {
    const component = create(<QuantityButton />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
