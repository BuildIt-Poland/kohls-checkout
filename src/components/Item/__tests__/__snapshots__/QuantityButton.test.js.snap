// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - QuantityButton renders QuantityButton component 1`] = `
.c0 {
  font-size: 3rem;
  font-weight: 700;
  background: none;
  box-shadow: none;
  border: 0;
}

<button
  className="c0"
/>
`;
